app/views/articles/edit.html.erb

```diff
@@ -31,3 +31,4 @@
 <%= render "articles/v2_form", article: @article, organizations: @organizations, version: @version %>
 
 <%= javascript_packs_with_chunks_tag "clipboardCopy", "articleForm", defer: true %>
+<div id="markdown-lint-js-path" class="hidden" data-markdown-it-js-path="<%= asset_path("markdown-it.min.js") %>" data-markdown-lint-js-path="<%= asset_path("markdownlint-browser.min.js") %>">
```

Comment:
would it be possible to move these two files in a vendor dir, like `vendor/assets/javascripts`? See https://guides.rubyonrails.org/asset_pipeline.html#asset-organization

Reply:
Good point! I've done that now - I had to add some additional config to `assets.rb` to get them precompile, but locally it seems to be working now